packages/backend/src/index.ts

```diff
@@ -262,12 +262,12 @@ passport.deserializeUser(async (id: string, done) => {
     }
 });
 
-export const slackService = new SlackService({
-    slackAuthenticationModel,
-    lightdashConfig,
-});
-
 if (lightdashConfig.scheduler?.enabled) {
+    const slackService = new SlackService({
```

Comment:
Is the slack bot now dependent on the scheduler? Can't a user have the slack bot without the scheduler?